nextcord/abc.py

```diff
@@ -1388,6 +1398,16 @@ async def send(
             A list of stickers to upload. Must be a maximum of 3.
 
             .. versionadded:: 2.0
+        flags: Optional[:class:`~nextcord.MessageFlags`]
+            The message flags being set for this message.
+            Currently only :class:`~nextcord.MessageFlags.suppress_embeds` is able to be set.
+
+            .. versionadded:: 2.4
+        suppress_embeds: Optional[:class:`bool`]
+            Whether to suppress embeds on this message.
+            This cannot be set with ``flags``.
```

Comment:
Looks like it can be set in conjunction though?

Reply:
I don't even see much point in `flags`, since if a flag isnt implemented, you can't really send it.

Reply:
it allows for future proofing I guess